patient-dashboard/frontend/scripts/deploy-with-vars.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Deployment script with environment variables
+
+# Source production environment if it exists
+if [ -f .env.production ]; then
+    export $(cat .env.production | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)